### PR TITLE
Convert pep-299 to rst.

### PR DESCRIPTION
**Pull request by Carreau**
As usual, the diff is not that big, but the rewrap of lines make it hard to vien on GitHub.

### CODE REVIEW

File: pep-0299.txt [hunk 1]
```diff
@@ -5,116 +5,125 @@ Last-Modified: $Date$
...
 
+
 Abstract
+========
 
-    Many Python modules are also intended to be callable as standalone
-    scripts.  This PEP proposes that a special function called
-    __main__() should serve this purpose.
+Many Python modules are also intended to be callable as standalone
+scripts.  This PEP proposes that a special function called `__main__()`
```
Comment (berkerpeksag) [inline pep-0299.txt]: \`\`\__main__()\`\`

File: pep-0299.txt [hunk 2]
```diff
@@ -5,116 +5,125 @@ Last-Modified: $Date$
...
+wishing to invoke the standalone functionality of different modules
+with different argument lists)
 
 
 Proposal
+========
 
-    The standard name of the 'main function' should be '__main__'.
-    When a module is invoked on the command line, such as
+The standard name of the 'main function' should be `__main__`. When a
```
Comment (berkerpeksag) [inline pep-0299.txt]: \`\`\__main__()\`\` (single backtick is not valid reST)

File: pep-0299.txt [hunk 3]
```diff
@@ -5,116 +5,125 @@ Last-Modified: $Date$
...
-        mymodule.__main__(['mymodule', ...])
+    import mymodule mymodule.__main__(['mymodule', ...])
 
-    It is up to mymodule to document thread-safety issues or other
-    issues which might restrict use of __main__.  (Other issues might
-    include use of mutually exclusive GUI modules, non-sharable
-    resources like hardware devices, reassignment of sys.stdin/stdout,
-    etc)
+It is up to ``mymodule`` to document thread-safety issues or other
+issues which might restrict use of `__main__`.  (Other issues might
```
Comment (berkerpeksag) [inline pep-0299.txt]: \`\`\__main__\`\`

File: pep-0299.txt [hunk 4]
```diff
@@ -5,116 +5,125 @@ Last-Modified: $Date$
...
+In ``modules/main.c``, the block near line 385 (after the
+``PyRun_AnyFileExFlags`` call) will be changed so that the above code
+(or its C equivalent) is executed.
 
 
 Open Issues
+===========
 
-    - Should the return value from __main__ be treated as the exit value?
+Should the return value from``__main__`` be treated as the exit
```
Comment (berkerpeksag) [inline pep-0299.txt]: I'd use the following markup here:

```rst
* Should the return value [...]

  Yes.  Many [...]
```

(notice indentation in the second line)

Comment (berkerpeksag) [inline pep-0299.txt]: Add a space after "value from".

File: pep-0299.txt (hunk 2 above)
Comment (Carreau) [inline pep-0299.txt]: > single backtick is not valid reST

Sorry habits of Markdown.

File: pep-0299.txt (hunk 4 above)
Comment (Carreau) [inline pep-0299.txt]: > (notice indentation in the second line)

I was unsure if I should use a list of a definition list which looks more like Question/Answer, but sure.

**Comment by Carreau** [comment]
Fix, rebased/ squashed.

**Comment by berkerpeksag** [comment]
Looks great, thanks! :)

**Pull request merged**